external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/internal/partition/KafkaPartitionReaders.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.storm.kafka.spout.internal.partition;
+
+import org.apache.kafka.common.PartitionInfo;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.storm.kafka.spout.KafkaSpoutStreams;
+import org.apache.storm.kafka.spout.KafkaSpoutStreamsNamedTopics;
+import org.apache.storm.kafka.spout.KafkaSpoutStreamsWildcardTopics;
+
+import java.util.HashSet;
+
+public final class KafkaPartitionReaders {
+    public static KafkaPartitionReader partitionReaderOf(KafkaSpoutStreams kafkaSpoutStreams) {
```

Comment:
these names are very misleading. Can you please call this create or newInstance?